.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   root: true,
   extends: [
-    '@react-native-community',
```

Comment:
do we know what changed there?

Reply:
The https://www.npmjs.com/package/@react-native-community/eslint-config was deprecated. It's last release was 2 years ago.

It's not mentioned on npm, but this PR makes the change https://github.com/facebook/react-native/pull/37347
The new package https://www.npmjs.com/package/@react-native/eslint-config is updated frequently.